### ISSUE TITLE
Take over Launchpad Repository/PPA

### ISSUE THREAD

**Issue by mpstark** [I-hate-farms/elementary-tweaks#9]
Hello!

I left you a comment on Reddit, but I don't think that you've seen it. I'd be prefectly willing to transfer ownership of the Launchpad repository (https://launchpad.net/elementary-tweaks) to you, which I believe that you could just mirror to your GitHub repository without a whole lot of effort, though I don't know the exact process.

Translations, which is another ticket, could be handled through Launchpad, as they are now.

**Comment by PerfectCarl** [comment]
Hello! 

thank you for your offer. I think that's indeed better if I take over the launchpad project and set up a github mirror (and close old issues).

What are the next steps?

**Comment by mpstark** [comment]
I just need your launchpad username, I think.

**Comment by PerfectCarl** [comment]
here you go https://launchpad.net/~name-is-carl

**Comment by mpstark** [comment]
You are now the official maintainer of the launchpad project, though now I cannot change you to the 'driver', because I'm not the maintainer anymore, probably should have set you as the driver first.

If you need anything, be sure to shoot me a reddit message or email (though again, terrible at email)

**Comment by quassy** [comment]
Great that it worked out.

@PerfectCarl, maybe you could update the description on Launchpad?

**Comment by Globegitter** [comment]
@PerfectCarl I am seeing a few open PRs for some time now, is this repo still maintained? It is great to have this on github now - maybe it is possible to maintain the repo here but just add some additional maintainers?

**Comment by embik** [comment]
@Globegitter I don't think this repository is maintained anymore, but there's [elementary-tweaks/elementary-tweaks](https://github.com/elementary-tweaks/elementary-tweaks). I kinda dropped out of that project because I stopped using elementary OS. But I'm sure @Philip-Scott would be grateful for additional maintainers.

**Comment by Globegitter** [comment]
@embik thanks for posting! I made a PR there :)
